modules/sandboxed-containers-prereqs-peer-pods-aws.adoc

```diff
@@ -0,0 +1,24 @@
+//Module included in the following assemblies:
+//
+// * deploying-sandboxed-container-workloads-peer-pods.adoc
+
+:_content-type: CONCEPT
+[id="sandboxed-containers-prerequisites-peer-pods-aws_{context}"]
+= Prerequisites for peer-pods using AWS
+
+If you are using AWS to create peer-pods, you must ensure the following requirements:
+
+* Your {product-title} cluster must be installed on AWS, with at least one worker node.
+* You have access to `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` credentials. These are used to create additional EC2 instances in the same VPC of the cluster.
+* You must have the AWS CLI tool installed and configured.
+* You must enable cluster communication on port 15150.
++
+Use the following command to enable internal cluster communication on port 15150:
```

Comment:
You can enable the port in the AWS/Azure (depending on section) console OR use the following...